lib/msal-angular/src/msal.guard.ts

```diff
@@ -79,9 +79,8 @@ export class MsalGuard implements CanActivate {
         /*
          * If a page with MSAL Guard is set as the redirect for acquireTokenSilent,
          * short-circuit to prevent redirecting or popups.
-         * TODO: Update to allow running in iframe once allowRedirectInIframe is implemented
          */
-        if (UrlString.hashContainsKnownProperties(window.location.hash) && BrowserUtils.isInIframe()) {
+        if (UrlString.hashContainsKnownProperties(window.location.hash)) {
```

Comment:
Sorry, I wasn't clear about the changes here. We want to keep the `isInIframe` check, but only if `allowRedirectInIframe` is `false`.
```suggestion
        if (UrlString.hashContainsKnownProperties(window.location.hash) && BrowserUtils.isInIframe() && !allowRedirectInIframe) {
```

I think for now I would revert this change, and we can talk about how to handle this incrementally, since may require some other work to expose the `allowRedirectInIframe` value.

Reply:
Okay makes sense. Reverting